harness/determined/_execution.py

```diff
@@ -3,11 +3,12 @@
 import os
 import pathlib
 import sys
-from typing import Any, Dict, Iterator, List, Optional, Tuple, Type
+from typing import Any, Dict, Iterator, List, Optional, Tuple, Type, cast
 
 import determined as det
 from determined import constants, core, gpu, load
 from determined.common import api
+from determined.pytorch import PyTorchTrial, PyTorchTrialContext
```

Comment:
two things.

First, [google import style guide](https://google.github.io/styleguide/pyguide.html#22-imports).  Just don't import bare symbols (except from typing).

Second, we must not import pytorch directly as part of importing determined.  Not all users will have torch installed.